.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+*       @marc-aurele-besner
+*       @mathilde-mi
+*       @Guillaume-web
+*       @BenoitGrilli
+*       @0xenj
```

Comment:
This is not valid, each line represent a file pattern, in this case * is use to represent ALL files, therefore all username need to be on the first line, separated by spaces https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners